src/style.css

```diff
@@ -1,3 +1,86 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  border: 0;
+}
+
+/*---------------------------------*/
+
+li {
+  padding: 0;
+  height: 60px;
```

Comment:
better to use vertical padding instead of hardcoding height

Reply:
If I do not use "height: 60px" for li, I have error in the tests